scripts/two_line_bash.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#
+#
+# This was casuing issues in tmux and always showed an "catkin_mil_ws", I'm sure that was intended
```

Comment:
When I went to full screen (control-b z) the prompt would expand to fill the whole screen. When I'd do the same command to exit full screen, it would keep the length of dashes the same which would make it like a 5 line prompt until I cleared the screen.